plugins/nsenter.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: nsenter
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/pabateman/kubectl-nsenter
+  shortDescription: "Execute shell command under container namespace of pod"
+  description: |
+    Kubectl plugin for pod's linux namespaces command execution via direct node ssh connection
+  caveats: |
+    Almost done. Now activate your ssh agent ang go on!
```

Comment:
This is not quite helpful, you can use this field to list some relevant caveats that could be show-stoppers to your plugin:

```
* This plugin needs SSH access to nodes.
* Your nodes need to have `nsenter` command.
```

Reply:
@ahmetb thanks for the feedback and review! Agree with all comments and corrected manifest according to it, with my own vision ofc :)